### ISSUE TITLE
I think I found a form of bug not sure how to get yalc to work here

### ISSUE THREAD

**Issue by erikswed** [wclr/yalc#172]
I can only get the first code changes after doing yalc add test-lib. After that there is no way to get any yalc push code changes to work.
I created  a [stackoverflowalso](https://stackoverflow.com/questions/69575534/how-to-make-code-changes-show-when-using-yalc)

steps(just normal steps):
- yalc publish in the test-lib
- I then run `yalc add test-lib` in my App or yalc link test-lib

When I then change the `test-lib` code and run `yalc push` then nothing happens.

Read on it getting stranger..

I even restart the App server and do the `yalc update test-lib` but the test-lib code changes is not visible. 

The strange thing is in my app VSCode node_modules I see the `test-lib` code changes. It's like the `yalc push` is propagating the changes to the app but the app is not using them 

It's even more strange is that when I restart VSCode and start the App Server the old `test-lib` code is stil used. 

I test to remove the `test-lib` with the command `yalc remove test-lib` and restart laptop and then do again `yalc add test-lib`. I then see the old code again.

I tried in Chrome Browser to do hard reset empty cashes but no luck getting this to work thinking that chrome was using old code cache .

When I set a breakpoint in Chrome browser like this:
![image](https://user-images.githubusercontent.com/3285249/137539945-e94b14a7-ab9b-4828-8ed7-e7e4f5712efa.png)

The very strange thing then happens that VSCode is stopping at the file and code since the first time I did `yalc add test-lib` , Since then I have done 20 `yalc push` and even publish . Where is VSCode getting this code from I'm buffeled.

When I go and look in yalc Store:
C:\Users\user\AppData\Local\Yalc\packages\notistack\1.0.10\src:

I see the latest code.
please help me out here this is crazy shit LOL

How can I make code changes show in Yalc?

  [1]: https://github.com/wclr/yalc
  [2]: https://i.stack.imgur.com/E1Ho7.png

**Comment by wclr** [comment]
> What about the package.json, maybe I must change something in that

I don't think so. You should understand the whole chain of tooling here, then you will be able to figure out what's wrong.
